frontend/src/pages/notebookController/NotebookController.tsx

```diff
@@ -68,7 +74,20 @@ export const NotebookController: React.FC = React.memo(() => {
       }
     };
     checkUserState().catch((e) => console.error(e));
-  }, [username, dashboardConfig, namespace]);
+  }, [username, dashboardConfig]);
+
+  const notebookNamespace = React.useMemo(
+    () => dashboardConfig.spec.notebookController?.notebookNamespace,
+    [dashboardConfig],
+  );
```

Comment:
This memo doesn't do anything. It will recompute anyways. This is just a de-structure of the `dashboardConfig`

Reply:
Updated.